irpc-iroh/src/lib.rs

```diff
@@ -89,11 +95,82 @@ mod multithreaded {
 }
 #[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
 use multithreaded::*;
```

Comment:
Use `n0_future::boxed::BoxFuture` for this.

Reply:
:+1: https://github.com/n0-computer/irpc/pull/9